source/includes/administration/_pricings.md

```diff
@@ -20,458 +20,761 @@ curl "https://cloudmc_endpoint/rest/pricings" \
+`unitPrice`<br/>*Map* | Map from currency code to unit price
+`deprecated`<br/>*Boolean* | True if priced product is deprecated
 
 <!-------------------- CREATE PRICING -------------------->
-#### Create pricing
+### Create Pricing
 
 `POST /pricings`
 
-Create a new pricing.
+Create a pricing and put a price on selected products.
 
 ```shell
-# Creates a new pricing
-curl -X POST "https://cloudmc_endpoint/rest/pricings" \
-   -H "MC-Api-Key: your_api_key"
+# Create a pricing
+curl -X POST "https://cloudmc_endpoint/v2/pricings" \
+   -H "MC-Api-Key: your_api_key" \
+   -H "Content-Type: application/json" \
+   -d "request_body"
 ```
-
 > Request body example:
 
-```js
+```json
 {
-	"organization": {
-		"id": "1d37a999-6ac6-4193-acb2-49df5c43ff31"
-	},
-	"productCatalogs": [
-		{
-			"id": "b6c89a59-34bf-4be4-a9bf-4414de84f5f7"
-		}
-	],
-	"name": {
-		"en": "name",
-		"fr": "nom",
-		"es": "nombre"
-	},
-	"description": {
-		"en": "desc_en",
-		"fr": "desc_fr",
-		"es": "desc_es"
-	},
-	"effectiveDate": "2020-05-06T00:00:00.000Z",
-	"pricingProducts": [
-		{
-			"product": {
-				"id": "c14daee2-4678-4710-b9af-fc26fbd3c7f3"
-			},
-			"cogs": 4,
-			"unitPrice": 5,
-			"currency": "CAD"
-		},
-		{
-			"product": {
-				"id": "c14daee2-4678-4710-b9af-fc26fbd3c7f3"
-			},
-			"cogs": 4,
-			"unitPrice": 5,
-			"currency": "USD"
-		}
-	],
-	"supportedCurrencies": [
-		"CAD",
-		"USD"
-	]
+      "name": {
+        "en": "Name here",
+        "fr": "Nom ici"
+      },
+      "description": {
+        "en": "Description here",
+        "fr": "Description ici"
+      },
+      "supportedCurrencies": [
+        "CAD"
+      ],
+      "effectiveDate": "2020-08-31T12:00:00Z",
+      "organization": {
+        "id": "52fd201e-aa82-4a27-86b3-ea9650a7fb1e"
+      },
+      "pricingProducts": [
+        {
+          "unitPrice": {
+            "CAD": 13
+          },
+          "product": {
+            "id": "dd3fcab9-5b31-4f08-9b50-ed3326bccfb4",
+          },
+          "cogs": {
+            "CAD": 10
+          },
+          "deprecated": false,
+        }
+      ]
 }
 ```
-> The above command return JSON structured like this:
 
-```js
+Required | &nbsp;
+---- | -----------
+`id`<br/>*UUID* | The id of the pricing
+`name`<br/>*Map* | A map from language to name value for that language
+`description`<br/>*Map* | A map from language to description value for that language
+`supportedCurrencies`<br/>*List* | A list of currencies supported by the pricing
+`effectiveDate`<br/>*Date* | The date at which the pricing will take effect.
+`pricingProducts`<br/>*Array<PricedProduct>* | Products with an associated price on it
+
+Priced product
+
+Required | &nbsp;
+---- | -----------
+`id`<br/>*UUID* | The id of the priced product
+`product`<br/>*Product* | The product being priced
+`cogs`<br/>*Map* | Map from currency code to cogs
+`unitPrice`<br/>*Map* | Map from currency code to unit price
+`deprecated`<br/>*Boolean* | True if priced product is deprecated
```

Comment:
seems odd to create a depreciated pricing?

Reply:
removed